src/storage/in_mem_csv_copier/in_mem_arrow_rel_copier.cpp

```diff
@@ -1,17 +1,18 @@
-#include "storage/in_mem_csv_copier/in_mem_rel_csv_copier.h"
+#include "storage/in_mem_csv_copier/in_mem_arrow_rel_copier.h"
 
 #include "spdlog/spdlog.h"
 #include "storage/in_mem_csv_copier/copy_csv_task.h"
 
 namespace kuzu {
 namespace storage {
 
-InMemRelCSVCopier::InMemRelCSVCopier(CSVDescription& csvDescription, string outputDirectory,
+InMemArrowRelCopier::InMemArrowRelCopier(CSVDescription& csvDescription, string outputDirectory,
```

Comment:
There are a few differences in this commit:
1. CopyConfig already exists in `configs.h`, so I renamed `CSVDescription` to `CopyDescription`.
2. Using `unique_ptr` leads to some errors, which indicates that it relates to calling the destructor of `CopyDescription`, I think it's because of using `std::move` on `CopyDescription` in the constructor of some other class when initializing. Using `shared_ptr` avoids the error.